css/about.css

```diff
@@ -0,0 +1,230 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Roboto", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-size: 18px;
+  margin-bottom: 15px;
+  font-family: "Rubik", sans-serif;
+}
+
+p {
+  line-height: 1.4;
+}
+
+.container-about-page {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.about-heading {
+  display: flex;
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+.about-heading nav a {
+  text-decoration: none;
+  margin-left: 70px;
+  color: black;
+}
+
+.about-heading nav a:hover {
+  color: red;
+  text-transform: uppercase;
+}
+.about-main {
+  margin-top: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+.about-main-body section {
+  border-bottom: 1px dashed black;
+  margin-top: 20px;
+}
+.about-main-body {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 20px;
+  justify-content: space-around;
+}
+.about-main-body h2 {
+  margin-top: 15px;
+}
+.about-main-body button {
+  padding: 5px 13px;
+  border-radius: 10px;
+  font-size: 12px;
+  margin: 15px 0 15px 0;
+  border: 2px solid black;
+  font-weight: bold;
+}
+.about-main-body button:hover {
```

Comment:
cool hover effects on the buttons. you can also add a transition property to the button so that on hover it has a duration in the transform instead of just happening immediately...for example adding transition: all 0.3s ease-in-out; to the button (not in the hover, in the buttons regular style parameters)